be/src/util/thread.cpp

```diff
@@ -397,8 +477,15 @@ Status ThreadJoiner::join() {
         }
         waited_ms += wait_for;
     }
-    return Status::Aborted(strings::Substitute("Timed out after $0ms joining on $1",
-                                               waited_ms, _thread->_name));
+    return Status::Aborted(
+            strings::Substitute("Timed out after $0ms joining on $1", waited_ms, _thread->_name));
 }
 
+void register_thread_display_page(WebPageHandler* web_page_handler) {
+    web_page_handler->register_template_page(
+            "/threadz", "Threads",
+            boost::bind(&ThreadMgr::display_thread_callback, thread_manager.get(),
```

Comment:
If we can use std::bind for the same purpose, It's better to replace it using C++ standard library.

Reply:
ok